dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -84,6 +106,75 @@ public static void main(String[] arguments) throws IOException, ClassNotFoundExc
     System.out.println(stringBuilder.toString());
   }
 
+  /**
+   * Parses arguments to get list of jar file paths.
+   *
+   * @param arguments command-line arguments
+   * @return list of the absolute paths to jar files, or null if arguments are invalid
+   * @throws RepositoryException when there is a problem in resolving the Maven coordinate to jar
+   */
+  @Nullable
+  static List<Path> parseArguments(String[] arguments) throws RepositoryException {
+    Options options = new Options();
+    options.addOption("c", "coordinate", true, "Maven coordinates (separated by ',')");
+    options.addOption("j", "jars", true, "Jar files (separated by ',')");
+
+    HelpFormatter formatter = new HelpFormatter();
+    CommandLineParser parser = new DefaultParser();
+    List<Path> jarFilePaths = new ArrayList<>();
+    try {
+      CommandLine cmd = parser.parse(options, arguments);
+      if (cmd.hasOption("c")) {
+        String mavenCoordinates = cmd.getOptionValue("c");
+        for (String coordinate : mavenCoordinates.split(",")) {
+          jarFilePaths.addAll(coordinateToJarPaths(coordinate));
+        }
+      }
+      if (cmd.hasOption("j")) {
+        String jarFiles = cmd.getOptionValue("j");
+        List<Path> jarFilesInArguments =
+            Arrays.stream(jarFiles.split(","))
+                .map(name -> (Paths.get(name)).toAbsolutePath())
+                .collect(Collectors.toList());
+        jarFilePaths.addAll(jarFilesInArguments);
+      }
+    } catch (ParseException ex) {
+      System.err.println("Failed to parse command line arguments: " + ex.getMessage());
+      formatter.printHelp("StaticLinkageChecker", options);
+      return null;
+    }
+    if (jarFilePaths.size() < 1) {
+      System.err.println("No jar files to scan.");
+      formatter.printHelp("StaticLinkageChecker", options);
+      return null;
```

Comment:
I might prefer to actually throw an exception here - we can't do anything useful in the calling function after this point. (same with the `return null` a few lines up too)

Reply:
Removed redundancy with raising RuntimeException